### ISSUE TITLE
Please add scrolls

### ISSUE THREAD

**Issue by ezloj** [Malgardian/TQCollector#31]
Scrolls are not in the list of items to be collected.

**Comment by Malgardian** [comment]
I never considered them collectables really, as you can just buy them at vendors anyway.

You're talking about scrolls like Divine Scroll of the Ice Nova, right?

**Comment by ezloj** [comment]
Yes. I think you are correct that they are as easy to get as going and buying. But that is still an item and you still need to spend gold on it. And it would be nice to see how many scrolls you have and if you actually have all of them

**Comment by Davb37** [comment]
And you could add potions, dyes (outfits) and quests items too, and make them optional.
I know it would be a lot of work for something so useless, but it can be good for people like me with compulsive collector issues :-D

**Comment by Malgardian** [comment]
Alright, I'll consider adding them. Might just make a config option to toggle them on or off.
